kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousTester.java

```diff
@@ -219,4 +220,26 @@ public interface KroxyliciousTester extends Closeable {
      */
     void close();
 
-}
+    /**
+     * Creates a single topic on the default Kafka cluster
+     * @param clusterName the name of the virtual cluster on which to create the topic
+     * @return the name of the created topic
+     */
+    default String createTopic(String clusterName) {
+        return createTopics(clusterName, 1).stream().findFirst().orElseThrow(() -> new IllegalStateException("Failed to create topic"));
+    }
+
+    /**
+     * Creates N topics with a fixed partition count and replication factor.
```

Comment:
Do we want to document partitions=1 and rf=1 here?

Reply:
Maybe, I saw them as internal implementation choices, and I'm not sold that they are the right defaults. 

E.g. I was wondering about changing the partition count to n`um Brokers` and rf to `num Brokers - 1`

Reply:
I think 1/1 is good enough for now.   They test can always add more replicas to a topic if need be, or use the admin client.

Reply:
In that case I'll add them to the docs.